integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ConversationService.java

```diff
@@ -72,7 +75,13 @@ public Conversation markFinished(Conversation conversation) {
         return repo.save(conversation);
     }
 
-    public void registerConversation(EDUCore message) {
+    public Conversation registerConversation(EDUCore message) {
+        Optional<Conversation> find = repo.findByConversationId(message.getId()).stream().findFirst();
+        if (find.isPresent()) {
+            log.warn(String.format("Conversation with id=%s already exists, not recreating", message.getId()));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Conversation with id=%s already exists, not recreating" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)